docs/guide/reproducing.md

```diff
@@ -0,0 +1,70 @@
+# Reproducing Paratimes
+
+If you wish to build paratime binaries yourself, you can use the
+environment provided as part of the SDK. This way you can also verify
+that the binaries match the ones running on the network.
+
+The steps below show how to build the test runtimes provided in the
+`oasis-sdk` sources; steps for other paratimes should be similar.
+
+## Environment Setup
+
+The build environment is provided as a Docker image containing all the
+necessary tools. Refer to your system's documentation for pointers on
+installing software.
+
+The runtime sources need to be mounted into the container so prepare a
+directory first, such as:
+
+```bash
+git clone https://github.com/oasisprotocol/oasis-sdk.git
+```
+
+## Running the Image
+
+The images are available in the `oasisprotocol/runtime-builder`
+repository on Docker Hub and are tagged with the same version numbers as
+releases of the SDK. To pull the image and run a container with it, run
```

Comment:
But there is no `v0.2.0` release :-)

Reply:
What will be the first actual tag?

Reply:
could use `master` for now, until there's a release that includes the runtime-builder? (since the `runtime-builder` builds a `master` docker image on every master merge)

Reply:
Ah yes, in that case let's change it to `master`.